library/ssl_tls13_generic.c

```diff
@@ -34,6 +34,16 @@
 #include "ssl_tls13_keys.h"
 #include "ssl_debug_helpers.h"
 
+#include "psa/crypto.h"
+#include "mbedtls/psa_util.h"
+
+const uint8_t mbedtls_ssl_tls13_hello_retry_request_magic[
```

Comment:
This seems unrelated to this commit. Probably an artefact of rebasing?

Reply:
Yes, it shows conflict. I'm trying to match the alignment, hopefully, that will solve that.